core/video/encoder.go

```diff
@@ -40,11 +40,20 @@ func init() {
 	}
 }
 
+// Checks to make sure an encoder has been set
+func GetEncoder() (string, error) {
+	if encoder == "" {
+		return encoder, fmt.Errorf("neither avconv nor ffmpeg was found")
+	} else {
```

Comment:
Drop the `else` - govet moans if the true block will always return and you have an `else`:

```go
func Encoder() (string, error) {
	if encoder == "" {
		return encoder, fmt.Errorf("neither avconv nor ffmpeg was found")
	}
	return encoder, nil
}
```